internal/querynode/query_collection.go

```diff
@@ -63,6 +64,9 @@ type queryCollection struct {
 
 	queryMsgStream       msgstream.MsgStream
 	queryResultMsgStream msgstream.MsgStream
+
+	schemaHelper *typeutil.SchemaHelper
+	vcm          storage.ChunkManager
```

Comment:
suggest keep the naming style as above, use "chunkManager"

Reply:
Have changed the type to *storage.VectorChunkManager, the name "vcm" is already clear.